app/controllers/spree/admin/adjustments_controller.rb

```diff
@@ -2,16 +2,18 @@ module Spree
   module Admin
     class AdjustmentsController < ::Admin::ResourceController
       belongs_to 'spree/order', find_by: :number
-      destroy.after :reload_order
 
       prepend_before_action :set_included_tax, only: [:create, :update]
+      before_action :set_order_id, only: [:create, :update]
+      after_action :update_order, only: [:create, :update, :destroy]
```

Comment:
this is getting a bit crazy IMO. I think it's time we implement these action methods and call `super` when needed. I would do it in a separate PR once we know this fixes the issue.

Reply:
Agreed, this whole controller needs some serious work.